deploy/ray-operator/config/samples/ray_v1_raycluster.bk.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-sample
+spec:
+  clusterName: raycluster-sample
+  images:
+    defaultImage: "reg.docker.alibaba-inc.com/volvo_ci_arcx/ray_package:a59ce20ec8b9876a67df84ca45dc456481bc2d3a"
+    clusterApiServerImage: "reg.docker.alibaba-inc.com/zeromem/clusterapiserver:20191008"
+  imagePullPolicy: "Always"
```

Comment:
need rollingUpdate strategy

Reply:
this file deleted. Strategy leaves user to choose